From 9e25ff620aed744fee3260ab2f22646bb80c790e Mon Sep 17 00:00:00 2001
From: unknown author <cooker@mandrivalinux.org>
Date: Thu, 11 Feb 2010 14:38:59 +0000
Subject: [PATCH 33/36] fix stack protector check

bogus check: -fstack-protector generates a warning and not an error if it's not supported
---
 configure.ac |    2 	1 +	1 -	0 !
 1 file changed, 1 insertion(+), 1 deletion(-)
---
 configure.ac |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.ac b/configure.ac
index d76425a..7da7652 100644
--- a/configure.ac
+++ b/configure.ac
@@ -41,7 +41,7 @@ if test "$GCC" = yes; then
     old_cflags=$CFLAGS
     echo
     for flag in $cflags_to_try; do
-        CFLAGS="$CFLAGS $flag"
+        CFLAGS="$CFLAGS $flag -Werror"
         AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[]], [[return 0;]])],[
                 echo "   $flag"
                 RPMCFLAGS="$RPMCFLAGS $flag"
-- 
1.6.4.4

